pkg/plugins/helm/kubernetes_handler.go

```diff
@@ -277,6 +277,21 @@ func (p *KubeHandler) DescribeClusterDetails(clusterWrapper *models.ClusterWrapp
 	return nil
 }
 
+func (p *KubeHandler) checkTillerIsExistedAndRunning(client *kubernetes.Clientset, credential, zone string) error {
+	deployCli := client.ExtensionsV1beta1().Deployments("kube-system")
```

Comment:
Read from global-config is better.

Reply:
What does this mean? kube-system read from global-config?